### PR TITLE
fix: Sign Verifiable Presentation bdd test bug

### PR DESCRIPTION
**Pull request by talwinder50**
Noticed a bug in 
`  await healthCheck(`${environment.HTTP_LOCAL_RESOLVER_URL}/` + did.id, 5000, "resolve did timeout!")`

This statement was added to check the availability of resolvable did.  But doesn't seem to work. 
There fore removed and added a loop to test sidetree did is available to resolve before signing presentation.

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
@talwinder50 this is a "fix:" rather than a "feat:". it would be helpful if you could update your commit description.

**Comment by talwinder50** [comment]
> @talwinder50 this is a "fix:" rather than a "feat:". it would be helpful if you could update your commit description (similar to the updated PR title).

Done

File: test/aries-js-worker/test/verifiable/verifiable.js [hunk 1]
```diff
@@ -133,6 +133,25 @@ async function verifiableStore(newAries, mode = wasmMode) {
         )
     })
 
+    it(modePrefix + "Alice makes sure that the DID is resolvable", async function () {
+        let success = false
+        for (var i = 0; i < 10; i++) {
```
Comment (soluchok) [inline test/aries-js-worker/test/verifiable/verifiable.js:138]: nit: there is a variable for  retries   `let retries = 10;`. Please use it in your test, and also fix it in the original test (from where you copied it).

File: test/aries-js-worker/test/verifiable/verifiable.js [hunk 2]
```diff
@@ -133,6 +133,25 @@ async function verifiableStore(newAries, mode = wasmMode) {
...
+                })
+                .catch(function(error) {
+                    console.log('will try to resolve again : attempt=',i+1);
+                });
+
+            if (success) {
+                break
+            }
+
+            await new Promise(r => setTimeout(r, 2000));
```
Comment (llorllale) [inline test/aries-js-worker/test/verifiable/verifiable.js:151]: can we sleep for less time? Our build's duration has gotten longer and longer...

Comment (troyronda) [reply test/aries-js-worker/test/verifiable/verifiable.js:151]: we should be looping around smaller timeouts.

File: test/aries-js-worker/test/verifiable/verifiable.js (hunk 1 above)
Comment (talwinder50) [reply test/aries-js-worker/test/verifiable/verifiable.js]: done

**Pull request merged**